README.md

```diff
@@ -1 +1,31 @@
-# protofolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HELLO MICROVERSE
```

Comment:
## [OPTIONAL]

- [ ] to make your `README` more professional and to not confuse visitors I would suggest you give your project an appropriate title.